### DISCUSSION TITLE
Custom To name in Admin Recipients field

### DISCUSSION THREAD

**Discussion by Christopholus** [solspace/craft-freeform#1433] [Feature Requests]
### What happened?

We have marketing automation that requires a specific To name to be sent, alongside of the email address.

When trying to set the following in the Admin Recipients field:
`{$AUTORESPONDER_TONAME} <email@emailprovider.net>`

We receive this error in the Freeform logs:
`Email "{$AUTORESPONDER_TONAME}" does not comply with addr-spec of RFC 2822 {"template":"generalContactAdmin","file":"/cms/vendor/symfony/mime/Address.php"} []`

Is there another way for us to specify a To name alongside of the email within Freeform?

Thank you in advance for the help!

### How can we reproduce this?

1. Create form
2. Set an admin recipient to have both a to name and email address
3. Submit form

### Versions

Freeform Pro 4.1.19
Craft Pro 4.10.6

**Comment by kjmartens** [comment]
Hi @Christopholus,

This is currently not possible with Freeform. I have noted this for future consideration. However, it would likely only happen on Freeform 5+ as it would likely not be very small of a change, at least not on Freeform 4.x, unfortunately.

Is there not any other way you can control the CMS automation? Surely you could have a check performed on another field value, etc? 🙂
